### PR TITLE
KIALI-3128/MAISTRA-628 - Adding Pull Secrets Option to Kiali Operator via Maistra Operator

### PR DESCRIPTION
**Pull request by gbaufake**
This Pull Request depends on https://github.com/kiali/kiali/pull/1247

### CODE REVIEW

**Comment by gbaufake** [comment]
cc @jmazzitelli

File: helm/istio/charts/kiali/templates/kiali-cr.yaml [hunk 1]
```diff
@@ -23,6 +23,9 @@ spec:
 {{- end }}
     image_name: "{{ .Values.hub }}/kiali"
     image_pull_policy: "{{ .Values.global.imagePullPolicy }}"
+{{- if .Values.global.imagePullSecrets }}
```
Comment (jmazzitelli) [inline helm/istio/charts/kiali/templates/kiali-cr.yaml:26]: where is the `{{- end}}` for this if-stmt?

File: helm/istio/charts/kiali/templates/kiali-cr.yaml [hunk 2]
```diff
@@ -23,6 +23,9 @@ spec:
...
+    image_pull_secrets: "{{ .Values.global.imagePullSecrets }}"
```
Comment (jmazzitelli) [inline helm/istio/charts/kiali/templates/kiali-cr.yaml]: here you set the value of the image_pull_secrets, but then you have a `{{ - range}}` on the next line with the same .Values.global.imagePullSecrets. That doesn't seem right - it looks like you are putting the secrets list in here twice?

**Comment by jmazzitelli** [comment]
I did not run this, but the template code looks correct.

**Comment by jmazzitelli** [comment]
@gbaufake You are going to need a Maistra team member to officially review/approve this to get it merged.

**Comment by jwendell** [comment]
Also, we should wait for 0.13 branch to be created and this PR should be targeting it. (This should happen very soon)

**Comment by gbaufake** [comment]
@jwendell 
@rcernich  updated to 1.0

I think it now ready to be reviewed

**Review by brian-avery — APPROVED** [review]

**Review by rcernich — APPROVED** [review]

**Pull request merged**